src/platform/icelake/include/platform/platcfg.h

```diff
@@ -37,4 +37,6 @@
 
 #define PLATFORM_MASTER_CORE_ID		0
 
+#define PLATFORM_LPSRAM_EBB_COUNT	1
```

Comment:
Looks like this should be a separate patch

Reply:
Do you mean separate commit? I can reorganize commits a bit and apply all platcfg.h changes before I do anything KPB related. Although it requires a bit of rework.